app/src/src/components/views/EventView.jsx

```diff
@@ -35,9 +36,12 @@ const EventView = ({ item, updateItem }) => {
     interested,
   } = item;
 
+  const userView = useContext(UserViewContext);
   return (
     <Card style={sharedItemViewStyles.container}>
       <Card.Content style={sharedItemViewStyles.cardContent}>
+      <TouchableOpacity style={styles.row}
+        onPress = { () => userView.setUser(creator) } >
```

Comment:
remove starting and trailing spaces ie { () => ... (creator) } (the ones beside the curly braces) for consistency